src/vos/evtree.c

```diff
@@ -1699,19 +1687,17 @@ evt_insert_or_split(struct evt_context *tcx, const struct evt_entry_in *ent_new,
 		if (mbr_update) {
 			D_ASSERT(nd_tmp != NULL);
 			mbr_update = false;
-			evt_mbr_read(&rtmp, nd_tmp);
-			mbr_changed = evt_node_rect_update(tcx, nd_cur,
-					   trace->tr_at, &rtmp);
+			mbr_changed = evt_node_mbr_update(tcx, nd_cur, nd_tmp,
```

Comment:
(style)  'nd' may be misspelled - perhaps 'and'?
(style)  'nd' may be misspelled - perhaps 'and'?